source/content/guides/drupal/drupal-hosted-createdashboard-set8/03-prepare.md

```diff
@@ -1,27 +1,27 @@
 ---
-title: Migrate a Site Created With the Pantheon Dashboard to Drupal 9
```

Comment:
@stovak , contains multiple references to D8. Should it say “prior version” instead?  See preview link: https://pr-8004--pantheon-docs.my.pantheonfrontend.website/docs/guides/drupal-hosted-createdashboard-set8/prepare